eng/scripts/Publish-Crates.ps1

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env pwsh
+
+#Requires -Version 7.0
+param(
+  [string]$PackagesPath,
+  [string[]]$CrateNames,
+  [string[]]$AdditionalOwners,
+  [string]$Token
+)
+
+$ErrorActionPreference = 'Stop'
+Set-StrictMode -Version 2.0
+
+function TryAddOwners($crateName) {
+  foreach ($owner in $AdditionalOwners) {
```

Comment:
If these are *additional* owners, where is the initial owner list? `azure-sdk` should always be an owner, for example.

Reply:
`azure-sdk` is the publisher and initial owner of any crate it publishes